### DISCUSSION TITLE
Fixed output name makes it impossible to deploy two pipelines to the same account and region

### DISCUSSION THREAD

**Discussion by donatoaz** [aws/aws-pdk#235] [General]
https://github.com/aws/aws-prototyping-sdk/blob/81cde33a61a32a1d80aaf867acb71bc56a3ecfc6/packages/pipeline/src/pdk-pipeline.ts#L172

The export name should be prefixable by a parameter in a way as to permit multiple deployments to the same account/region.

As it is today, trying to deploy a second PDKPipeline will fail with the error:

```
Export with name CodeRepositoryGRCUrl is already exported by stack PipelineStack. Rollback requested by user.
```

**Comment by cogwirrel** [comment]
Thanks for pointing this out! Agreed it would be nice not to hardcode the export name! Perhaps we could prefix with the `repositoryName` since that's already an input that would need to be unique per `PdkPipeline`? :)

I've opened this as an issue here: https://github.com/aws/aws-prototyping-sdk/issues/239
